src/app/modules/datagrid/datagrid-controller.js

```diff
@@ -143,13 +143,20 @@
 			$scope.getBooks();
 
 			var filterTags = function(searchTerm, cellValue, row, column) {
```

Comment:
Basically, this change makes it possible to filter on multiple tags at the same time.  Though not technically specific to making system tags work, Grant made the change here because he realized when implementing system tags that a user might want to filter within system tags.

I can add some comments.